src/app/recent/[...regionCodeSlug]/page.tsx

```diff
@@ -149,20 +149,21 @@ async function BirdCard({ sighting }: { sighting: Sighting }) {
 
 export default async function RecentSightingsPage({
   params,
+  searchParams,
 }: {
   params: {
     regionCodeSlug: string[];
   };
+  searchParams: { name: string };
```

Comment:
Made use of [searchParams](https://nextjs.org/docs/app/api-reference/file-conventions/page) instead of useSearchParams. To use useSearchParams requires page.tsx to be a client component, and this caused strange behaviour where the home page Links and Top-nav components would not work.

Reply:
Yep great shout, I think we should keep as much stuff running on the server as possible and defer things to the client only when absolutely necessary.

Reply:
Good to know!